diff --git a/html/controllers/ui_data/summary3.php b/html/controllers/ui_data/summary3.php
index c2e93129..f791e6ac 100644
--- a/html/controllers/ui_data/summary3.php
+++ b/html/controllers/ui_data/summary3.php
@@ -47,7 +47,9 @@ try {
             }
         }
     }
-
+/*
+ *  Removed when there is no JOBS REALM
+ *
     $query_descripter = new \User\Elements\QueryDescripter('Jobs', 'none');

     $query = new \DataWarehouse\Query\Jobs\Aggregate($aggregation_unit, $start_date, $end_date, 'none', 'all', $query_descripter->pullQueryParameters($raw_parameters));
@@ -67,6 +69,7 @@ try {
             throw $e;
         }
     }
+*/
     $mostPrivilegedAcl = Acls::getMostPrivilegedAcl($logged_in_user);

     $rolesConfig = \Configuration\XdmodConfiguration::assocArrayFactory('roles.json', CONFIG_DIR);
@@ -152,4 +155,3 @@ try {
                            'data' => array(),
                            'success' => false));
 }
-
